build/testBuildImages.sh

```diff
@@ -23,6 +23,20 @@ else
     $buildBuildImagesScript "$@"
```

Comment:
we're now passing new arguments to this script, but we still have this `$@` which will capture all arguments and pass to $buildBuildImagesScript. Will it work when `skipBuildingImages` is not passed in and this else executes?

Reply:
I'm not sure if we need to pass all of the arguments to `buildBuildImages.sh` -- looking at that script, the only place I see it using arguments provided to the script is the following:

```
declare -r REPO_DIR=$( cd $( dirname "$0" ) && cd .. && pwd )
```

which is the same line that the `testBuildImages.sh` script has. I think we should change this call to be `$buildBuildImagesScript "$0"`.